device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-D108C8/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,71 @@
+{%- set default_cable = '5m' %}
+{%- set default_speed = '50G' %}
+
+{%- macro generate_port_lists(PORT_10G, PORT_25G, PORT_40G, PORT_50G, PORT_100G) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,12) %}
+        {%- if PORT_50G.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
+        {%- if PORT_50G.append("Ethernet%d" % (port_idx * 4 + 2)) %}{%- endif %}
+    {%- endfor %}
+    {%- for port_idx in range(20,64) %}
+        {%- if PORT_50G.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
+        {%- if PORT_50G.append("Ethernet%d" % (port_idx * 4 + 2)) %}{%- endif %}
+    {%- endfor %}
+    {%- for port_idx in range(12,20) %}
+        {%- if PORT_100G.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "32799936",
+            "type": "ingress",
+            "mode": "dynamic",
+            "xoff": "8356608"
+        },
+        "egress_lossy_pool": {
+            "size": "26240448",
+            "type": "egress",
+            "mode": "dynamic"
+        },
+        "egress_lossless_pool": {
+            "size": "42349632",
+            "type": "egress",
+            "mode": "static"
+        }
+    },
+    "BUFFER_PROFILE": {
+        "ingress_lossless_profile_50g": {
```

Comment:
Hi Ying, BUFFER_PROFILE section should have default values and not specific to any speed isn't?